### PR TITLE
Add Build process Travis

### PR DESCRIPTION
**Pull request by Andries-Smit**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+  - MX_ENVIRONMENT=Sandbox
+  - MX_PROJECT_ID=8648ccda-9281-4768-abb2-8ce61a80e2f1
+  - MX_USER=ci@flockofbirds.nl
+  - secure: fzIxF5ER2BZ55xcGTzLML9vAp0t7sVIIDrvJaTcX1VOkPk6pWwWzFQFT5o/8cN9W/XEQfWWkbyfL1rp9+WsE9DgGA7NK3lfSIhC87cxHT9mig9EO5zCOX9Chtxe+wIvfN/dAGFhvr3SOBr5JMn3BvwPGJXdpStK4+ehZjbBaX73odQdT3aa4MYUc9tH3JFMpcORU2WlpqfJogn03BcXM1Qkxh6gNEB+1Q7XraNIrJUsybw8NDrmZejYLCIJYSAWTXfK8tD3QUZDFqOB0j2MVSr1RBy9FXA6TYgYamRPJmzPp65rC94x30KqbOnRbD97SKnSZ887lGnA0s+dhrOqcGTXpdvPAb4YY7QUHEyNRy68CXBv8F4yxl/BTBG5MgHLHUoHWvUu5nJ0axh9SPIeloglpckEEUCoo9ZW5QBcLtKHgi0GN0wPjKS7n+EDq94ZoA2j+V+lhD7uwgnRvIrFVVqEacJPxB7yI+EX0mUkiuVr0+fus20u+JCQ96eCSfZLZhLRKMSx+Jiqk/lRjlgMKwY0vPP8827FySby72nxg8jaMNbCC6/uTG77l81cshoejCPygahiRQ261dUswDr0/XDoihss2vp4NUKWGswGjo9HRQ2RyIC/gwUIUsKL4lIW2Of9utQA/T2bsZWlnhL3TrxTtM4uCD9ypWSZcOWVafIM=
```
Comment (Andries-Smit) [inline .travis.yml:32]: Remove defaults

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,42 @@
 sudo: required
 dist: trusty
```
Comment (Andries-Smit) [inline .travis.yml:2]: Work on trusty
https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming?utm_source=web&utm_medium=banner&&utm_campaign=trusty-default

File: .travis.yml [hunk 3]
```diff
@@ -1,19 +1,42 @@
...
+deploy:
+  provider: releases
+  api_key:
+    secure: mQtgwYHfLN8JGuTUQfKrBxJsGw5oG9ii0qa8gBDQL5uz2XHxj7BYzuUfbu9P0emWp4vWImLaIp398GM1EicoBvCSSlJrfW6KnlNz+VT3c93wm1SVf+Vjk6Pm5Qjnyk43R11L0xhSumxPk+BWycdQ0yaNmmjRLriVbEz2d9JnHMcdYBRXcIC0HrKFgS8KEiuPzxaHcqALOVzczGuVe3sCHhDZ0pmlpQlu+zzBLjN18UbS2H3lx7wGg5pcKzKuuo2DxxZdLiMGCbh3bfBaWwCBHZ3nDESSj4UbZyhPmVCYijNRLauYQJgg2wDO3g5lD3UNIZ5Hx7zMUX1h0b+DPCaAX7wsuxgHTF243Nx+Qcp5R/FEKqNHYRqYFnhHfvx+ABEaZEt5xAqlSV44LhQuW+DAA/X/3+/BZSOx7lO+yLkEWDkKaOwzLcLMue0eBw2TZqciXvrypzz3JVcx/d0VeSbSnmmAT7eTRxdbxuFxCMc+tPw59kL+hbG7Rq4I/wWjtoi4AuEALn++au4Wm5zIkQmpMoCFNJ9UwWHas4SCDh140sGMYj8DJC7YghWW2SnHoOhFzpgMtOvoptpN2c5rjFBhOt3rIlsIE1TjlPztihSH69rd2cm3FgM7iljjdd3E6XwrXC5mTMWpcOfTrk4aR8xJnIqQNmPmUusG5q9ZnxBcmRw=
+  file_glob: true
+  file: dist/release/*
+  skip_cleanup: true
+  on:
+    repo: Andries-Smit/rich-text
```
Comment (Andries-Smit) [inline .travis.yml:41]: Repo should not be bound

File: .travis.yml (hunk 1 above)
Comment (edwinmp) [reply .travis.yml:32]: Done.

File: .travis.yml (hunk 2 above)
Comment (edwinmp) [reply .travis.yml:2]: Done.

File: .travis.yml (hunk 3 above)
Comment (edwinmp) [reply .travis.yml:41]: Removed.

**Pull request closed without merging**